config/locales/devise.en.yml

```diff
@@ -0,0 +1,33 @@
+en:
+  devise:
+    janrain_widget_url:
+      '//cdn.oreillystatic.com/members/js/widgets/login-widget.min.js'
+  capturable:
+    widget:
+      greeting: "Hi "
+      sign_in_up: Sign In/Sign Up
+      sign_out: Sign Out
+    noscript:
+      title: There was a problem loading the login page.
+      enable_javascript: Please enable JavaScript to sign in.
+      description_html: |-
+        <p>Please check your browser settings for these possible blockers, and try again once you've made any necessary changes.
+        Feel free to email <a href="mailto:accounts@oreilly.com">accounts@oreilly.com</a> if you have questions.</p>
```

Comment:
Yeah, I think that buys us a little, because we wouldn't have to update both apps to change the email address. I'll extract it to another translation key, so it can be overridden in the host app.